dev/manual_tests/raw_keyboard.dart

```diff
@@ -49,28 +49,29 @@ class _HardwareKeyDemoState extends State<RawKeyboardDemo> {
         onTap: () {
```

Comment:
The reason I called it "at" was so you could do `if (Focus.at(context)) ...`, so it's sad to see you factor that out into a separate variable. :-P